frontend/lib/js/category-trees/actions.js

```diff
@@ -90,3 +92,8 @@ export const loadTree = (datasetId: DatasetIdType, treeId: TreeNodeIdType) => {
       );
   };
 };
+
+export const searchTreesStart = (searchStr: string) =>
+  ({type: SEARCH_TREES_START, payload: { searchStr }});
+export const searchTreesEnd = (searchStr: string, result: any) =>
```

Comment:
This isn't used anywhere (yet), is it? 

Maybe you should move the search result computation into a redux thunk (although there's nothing asynchronous going on at the moment). 

It's nice that we already have the prerequisites for computing the search results on the backend, but we should make sure that we don't forget about this action type by using it somewhere.